doc/classes/NavigationMesh.xml

```diff
@@ -28,6 +28,7 @@
 			<argument index="0" name="mesh" type="Mesh" />
 			<description>
 				Initializes the navigation mesh by setting the vertices and indices according to a [Mesh].
+				Note that the given [Mesh] must be of type PRIMITIVE_TRIANGLES, and have an index array.
```

Comment:
Stylistic change:

```suggestion
				[b]Note:[/b] The given [code]mesh[/code] must be of type [constant Mesh.PRIMITIVE_TRIANGLES] and have an index array.
```

Also, the note should describe what happens if this condition is not met. Is an error message printed? If not, we should add that.

Reply:
Thanks for helping out with the formatting! :) This is my first PR here so I'm not quite sure what I'm doing! ^^ 
Also - if this condition is not met the result will indeed be a silent failure. I found this information [here](https://github.com/godotengine/godot/blob/36fd7f213d0082f4465cb0c89f7f9f58d06f8a0b/scene/resources/navigation_mesh.cpp#L48), and as you can see, the mesh surfaces that are not PRIMITIVE_TRIANGLES or lack an index array are just skipped.